src/app/components/shared-components/addons-primary/drawerNew.tsx

```diff
@@ -0,0 +1,103 @@
+import {
+  Drawer,
+  DrawerActions,
+  DrawerCloseButton,
+  DrawerContent,
+  DrawerContentBody,
+  DrawerHead,
+  DrawerPanelContent,
+  Progress,
+  ProgressSize,
+  ProgressMeasureLocation,
+  ProgressVariant,
+  Button,
+  Text,
+  TextVariants,
+} from "@patternfly/react-core"
+import {
+  Table,
+  TableHeader,
+  TableBody,
+  TableText,
+  TableVariant,
+} from "@patternfly/react-table"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { faThumbsUp, faThumbsDown } from "@fortawesome/free-solid-svg-icons"
+import React, { useState } from "react"
+
+const TextTable = (data: string) => {
+  return <TableText wrapModifier="wrap">{data}</TableText>
+}
+
+const ProgressBar = (data: number) => {
+  const title = String(data).concat("%")
+  return (
+    <Progress
+      value={data}
+      title={title}
+      size={ProgressSize.sm}
+      measureLocation={ProgressMeasureLocation.none}
+    />
+  )
+}
+
+const FeedbackButtons = () => {
+  return (
+    <div>
+      <Button variant="link" icon={<FontAwesomeIcon icon={faThumbsUp} />} />
+      <Button variant="link" icon={<FontAwesomeIcon icon={faThumbsDown} />} />
+    </div>
+  )
+}
+
+const DrawerFC = () => {
```

Comment:
What happened to the drawerStack that i had created?Why is that not reused?I feel that should have been reused as the drawer development for that is also done.

Reply:
i didnt want to mess up your code so i created a new file you can take this code and modify yours and delete this file think of it as a temp code

Reply:
Sure.